packages/react-query/package.json

```diff
@@ -64,14 +64,14 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "@tanstack/react-query": "^5.40.0",
+    "@tanstack/react-query": "^5.49.2",
```

Comment:
nice

Reply:
fuck I hate that manypkg bumps peerDep range. We can keep it at 5.40 assuming most ppl wont use this RSC void prefetch thing?

Reply:
i think we should bump it and i'm fine with bumping this with any minor/patch release